kubernetes/namespaces/web/site/deployment.yaml

```diff
@@ -13,6 +13,19 @@ spec:
       labels:
         app: site
     spec:
+      initContainers:
+        - name: migrations
+          image: ghcr.io/python-discord/site:latest
+          imagePullPolicy: Always
+          command:
+            - sh
```

Comment:
Why do we need a shell? Is it to resolve `poetry` on PATH?

Reply:
Sanity check complete. I am insane.

Reply:
Amended in 549a559